packages/components/src/List/Toolbar/ColumnChooser/Content/DefaultColumnChooser.components.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import theme from './ColumnChooser.scss';
+import ActionButton from '../../../../Actions/ActionButton';
+import ColumnDisplayer from '../ColumnDisplayer';
+
+export const DefaultHeader = ({ t }) => (
+	<div className={classNames(theme['tc-column-chooser-header'], 'tc-column-chooser-header')}>
+		{t('COLUMN_CHOOSER_HEADER_TITLE', {
+			defaultValue: 'Modify columns position',
+		})}
+	</div>
+);
+
+DefaultHeader.propTypes = {
+	t: PropTypes.func.isRequired,
+};
+
+export const getColumnDisplay = (length, onChangeVisibility, onBlurOrder, onKeyPressOrder, t) => (
+	column,
+	index,
+) => {
+	const displayerProps = {
```

Comment:
You don't need this object

Reply:
I thought that was more readable, but ok